documentation/docs/04-hooks.md

```diff
@@ -116,3 +116,21 @@ export async function serverFetch(request) {
 	return fetch(request);
 }
 ```
+
+### handleError
+
+This function allows you to handle errors with your own error handler.
+
+For example, you may have a custom logging scheme that you send to an external system, and you want to format the error in a manner consistent with your other logs.
+
+```js
+/** @type {import('@sveltejs/kit').HandleError} */
+export async function handleError({error}) {
```

Comment:
can we go ahead and add the `request` as a parameter here as well? I want to make sure we have access to it everywhere `handleError` is called and there wouldn't be any issues with this design. I think it should be straight-forward, but just want to confirm

Reply:
I like that idea. I'll give it a shot

Reply:
we should add `request` to the docs